pam/integration-tests/helpers_test.go

```diff
@@ -31,9 +31,10 @@ import (
 )
 
 var (
-	authdTestSessionTime  = time.Now()
```

Comment:
I think you meant "independent of whether the test failed or not" in the commit message, not "dependent" (because it already was dependent on that before)